flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointBriefComputerContext.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.checkpoint;
+
+import org.apache.flink.runtime.concurrent.ScheduledExecutor;
+
+/** Provides the context for {@link CheckpointBriefComputer} to compute the brief of checkpoints. */
+public interface CheckpointBriefComputerContext {
+
+    /**
+     * Acquires the main thread executor for this job.
+     *
+     * @return The main thread executor.
+     */
+    ScheduledExecutor getMainExecutor();
```

Comment:
I'm a bit concerns here since that`CheckpointCoordinator` uses `executor` to refer to the IO executor in JM side, would it be better if we distinguish them explicitly here ?